next-tick-browser.js

```diff
@@ -0,0 +1,8 @@
+module.exports = nextTick;
+
+function nextTick(fn) {
+  var args = Array.prototype.slice.call(arguments, 1);
+  setTimeout(function () {
+    fn.apply(null, args);
+  }, 0);
+}
```

Comment:
This is non-breaking for webpack users, right? But potentially breaking for `browserify` users.

Reply:
> This is non-breaking for webpack users, right?

Yes, it's a slightly modified version of the webpack's polyfill.

> But potentially breaking for `browserify` users.

Well, potentially yes, but I don't think they will face any bugs. The only difference between these implementations is that the browserify's version drains queue synchronously on subsequent calls to `nextTick`.

Reply:
have you got a link to Browserify impl? I think Browserify version might be slightly faster and provide a better experience for users overall.

Reply:
https://github.com/defunctzombie/node-process/blob/master/browser.js#L90-L154

Reply:
I prefer the browserify one, and more importantly this was designed to work with browserify first.

Reply:
`test/browser/test-stream2-unpipe-drain.js` fails if I use browserify's version of `nextTick`.

I think it happens because test runner uses it's own version of `process.nextTick` polyfill (which comes from browserify) and `readable-stream` uses our own version so we end up with two different queues of async events.

webpack's version doesn't have this problem.

Reply:
I've pushed a version with browserify's `nextTick`. If you want, you can try to fix this test - I don't have any more time to spend on it.